### PR TITLE
data: deepak039543

### PR DESCRIPTION
**Pull request by deepak039543**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: deepak039543 [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "RAM JI MISHRA",
+  "bio": "I am a 2nd Year student of RAI UNIVERSITY.I like to contribute in Open Source Projects."
```
Comment (sandramsc) [inline deepak039543:3]: There's a missing comma (**,**) at the end of line 3.
```suggestion
  "bio": "I am a 2nd Year student of RAI UNIVERSITY.I like to contribute in Open Source Projects.",
```

**Review by sandramsc — CHANGES_REQUESTED** [review]
Hi Ram, I've added an inline comment with a suggested change. Also in future, consider perhaps adding more links, testimonials and milestones to your profile. With that said....
wlecome2biodrop

**Review by kumarsonsoff3 — DISMISSED** [review]
Please add the file extension `.json` in the end of the filename, [rename here](https://github.com/deepak039543/BioDrop/edit/patch-1/data/deepak039543).

**Comment by sandramsc** [comment]
> Please add the file extension `.json` in the end of the filename, [rename here](https://github.com/deepak039543/BioDrop/edit/patch-1/data/deepak039543).

Good spot! 🙂

**Review by amandamartin-dev — APPROVED** [review]
Looks good! Thank you for addressing all comments quickly!

**Review by SaraJaoude — APPROVED** [review]
If you wish to add more links or any other features, once you are ready please do check our [Documentation](https://www.biodrop.io/docs).  You can choose whether any edits/additions are made through JSON or our Forms.

I will merge your PR now and would encourage you to check out the Create & Manage Profile Dashboard on the BioDrop app so that you can get an idea of how you can customise it further.

**Pull request merged**